### ISSUE TITLE
Create preview for dev branch

### ISSUE THREAD

**Issue by holanita** [Green-Software-Foundation/green-software-patterns#174]
adding this quickly as I'm short on time. to be completed later.

Bill asked for a preview of the dev branch, something like dev.greensoftware... (Asim will figure this out)

**Comment by holanita** [comment]
@zanete is [PATT] an abbreviation for patterns? Sorry I wasn't aware. Have you / Osama picked this up already?

**Comment by zanete** [comment]
@holanita I only added it to things that are in web or other discipline boards so that it's easier to understand what it's about at a glance when mixed in with other projects. If you have another abbrev that you're using we can switch to that.

**Comment by holanita** [comment]
No that's perfectly fine. I just thought it was me and I added those letters by mistake that's why I removed them last time :)

**Comment by zanete** [comment]
@holanita no, it's not releated. Do we still need to do this? It's also been assigned to @jawache and we weren't looking into it. Let me know if you are happy to close?
